cluster-autoscaler/utils.go

```diff
@@ -119,24 +132,52 @@ func NewNodeLister(kubeClient *kube_client.Client) *ReadyNodeLister {
 	}
 }
 
-// GetNewestNode returns the newest node from the given list.
-func GetNewestNode(nodes []*kube_api.Node) *kube_api.Node {
-	var result *kube_api.Node
+// GetNewestNodeTime returns time when the newest node became available for scheduler.
+// TODO: This function should use LastTransitionTime from NodeReady condition.
+func GetNewestNodeTime(nodes []*kube_api.Node) time.Time {
+	var result time.Time
 	for _, node := range nodes {
-		if result == nil || node.CreationTimestamp.After(result.CreationTimestamp.Time) {
-			result = node
+		if node.CreationTimestamp.After(result) {
+			result = node.CreationTimestamp.Time
+		}
+	}
+	return result.Add(1 * time.Minute)
```

Comment:
How about adding 1 minute somewhere else and making it constant? As for now the function GetNewestNodeTime doesn't return what its name declares.